packages/eslint-plugin/src/configs/project.ts

```diff
@@ -0,0 +1,184 @@
+/**
+ * ESLint configuration used by the Openverse project itself.
```

Comment:
I don't quite understand what "used by the Openverse project itself" means here. At first I thought it's for the top-level files, or maybe for the automations etc, but then I saw rules that are for `vue`.

Reply:
Updated in the latest commits.